backend/controllers/posts-controller.js

```diff
@@ -135,8 +135,11 @@ export const updatePostHandler = async (req, res) => {
     if (!updatedPost) {
       return res.status(HTTP_STATUS.NOT_FOUND).json({ message: RESPONSE_MESSAGES.POSTS.NOT_FOUND });
     }
-
-    res.status(HTTP_STATUS.OK).json(updatedPost);
+    // invalidate the redis cache
+    await deleteDataFromCache(REDIS_KEYS.ALL_POSTS),
+      await deleteDataFromCache(REDIS_KEYS.FEATURED_POSTS),
```

Comment:
It should ideally format, I'm confused, why isn't it working for you !!
we have a husky hook which actually formats the code, 
Isn't it working for you?

Reply:
i guess its not working for me.
i will manually do "npm run format" in both frontend and backend and push the code

Reply:
Sure @devsharmagit